src/community/geogig/src/main/java/org/geogig/geoserver/security/SecurityLogger.java

```diff
@@ -125,15 +125,15 @@ private void pre(AbstractGeoGigOp<?> command) {
     private String repoUrl(AbstractGeoGigOp<?> command) {
         Context context = command.context();
         if (context == null) {
-            return null;
+            return "null";
```

Comment:
This seems inherently wrong, and likely to cause trouble elsewhere - is there any reason we would be returning the string `"null"` instead of the actual value `null` or even an empty string?